eclipse-formatter.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I guess we need to the license header in this file similarly as in `checkstyle.xml`.

Reply:
Agree.

Reply:
OMG, I omitted it in the last update. :sweat: